app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -203,6 +203,7 @@ def self.params_for_create
                         :id         => "authentications.bearer.auth_key",
                         :name       => "authentications.bearer.auth_key",
                         :label      => "Token",
+                        :helperText => _('Note that the Default, Metrics, and Alert Endpoints, if enabled, must be revalidated every time the token is changed or updated'),
```

Comment:
Looks good to me - another option:

```suggestion
                        :helperText => _('NOTE: The Default, Metrics, and Alert Endpoints, if enabled, must be revalidated every time the token is changed or updated'),
```

Reply:
@dehawkins512 or @rwellon can you give suggestions for wording here, or if it's ok give it a :+1: ?